cdk/bin/app-config-app.ts

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
```

Comment:
Let's add the shorter license header as in:

```javascript
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: MIT-0
```

Reply:
Added in e9d4fff